cookbook/entities/acl.rst

```diff
@@ -0,0 +1,246 @@
+How to Protect Resources Using ACLs
+===================================
+
+Using ACLs you can granularly grant access to your entities. Doing so requires three steps:
+
+#. :ref:`Activate ACL checks for your entities <coobook-entities-acl-enable>`.
+
+#. :ref:`Create access control lists for all available actions <coobook-entities-acl-create>`.
+
+#. :ref:`Add access checks<coobook-entities-acl-check>` to where your entities are displayed or
+   manipulated.
+
+.. _coobook-entities-acl-enable:
+
+Activating ACL Checks on your Entities
```

Comment:
You have no say anything about ownership type configuration and what developer can limit collection of available permissions for entity.